src/main/java/com/almostreliable/ponderjs/api/ExtendedSceneBuilder.java

```diff
@@ -89,6 +94,34 @@ public void encapsulateBounds(BlockPos size) {
         });
     }
 
+    public void playSound(SoundEvent soundEvent, SoundSource soundsource, float volume, float pitch) {
+        Level clientlevel = Minecraft.getInstance().level;
+        Player clientplayer = Minecraft.getInstance().player;
```

Comment:
Please rename these fields to either `level` and `player` without the `client` prefix or make it camelCase so it is `clientLevel` and `clientPlayer`.